src/types/Theme.ts

```diff
@@ -3,6 +3,9 @@ import { asNumber, asObject } from 'cleaners'
 export type ImageProp = { uri: string } | number
 
 export interface ThemeDot {
+  // We the dot color with the accent color, we want to keep the dot color,
+  // or drop the dot:
+  accent?: 'keep' | 'drop'
```

Comment:
Hmm. I won't fight back hard on this but this is another bizarre api at is SO specific to our current design. I would have still preferred a simple accent array that allows any of the dots to be changed programmatically after the Theme is applied.